ding/policy/dqn.py

```diff
@@ -248,6 +248,8 @@ def _forward_learn(self, data: List[Dict[str, Any]]) -> Dict[str, Any]:
         .. note::
             For more detailed examples, please refer to our unittest for DQNPolicy: ``ding.policy.tests.test_dqn``.
         """
+        set_noise_mode(self._learn_model, True)
```

Comment:
use noisy_net to control this line

Another question: how to deal with target_model in noisy net